.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
```

Comment:
I did not know about this trigger, handy!

Reply:
Oh, yeah, this is here because I did a copy and paste from a personal repo. In https://github.com/rbreslow/docker-garrysmodds/pull/2, I set up a Zapier integration to follow an RSS feed and dispatch the GitHub Actions workflow on-demand.

It should be safe to leave it in, and might actually be helpful if we approached #19. I think this flag just _allows_ manual dispatching of workflows.

Reply:
That matches my understanding. I think being able to manually trigger it could be useful.